### ISSUE TITLE
CSRF issue when adding an image to speaker

### ISSUE THREAD

**Issue by rknall** [pretalx/pretalx#1774]
### Problem and impact

When a user has submitted a talk on https://conference.wireshark.org/sharkfest-24-eu/ and tries to upload an image for this speaker profile (does not matter if he does it during submission or afterwards), we get a CSRF issue (error 403). Same happens, when the issue is triggered through the admin site.

### Expected behaviour

The behavior would be to upload the image without any issues. The image was neither too big nor wrongly formatted.

### Steps to reproduce

1. Create a user profile
2. Upload an image
3. -> Error happens

### Screenshots

![image](https://github.com/pretalx/pretalx/assets/1235641/30b80d3f-305a-49b3-9466-843546646ed9)

### Link

https://conference.wireshark.org/sharkfest-24-eu/

### Browser (software, desktop or mobile?) and version

Safari / Chrome

### Version

v2023.2.0.dev0

**Comment by rixx** [comment]
Honestly, I don't know what kinds of logs would help. Both the development setup and pretalx.com have no issues with image uploads, so it's highly likely that there's something off with your setup, and that this is not a pretalx bug. Have you made sure to forward/set all the headers listed in the documentation?

**Comment by rknall** [comment]
Yes we have. The last thing we are checking right now is, if the cloudflare instance does interfere with it. That might be an issue still. Good to know though that it did not happen in the past on your end, helps further narrowing down the issue
